packages/sdk/src/lib/types/coreTypes.ts

```diff
@@ -267,6 +267,11 @@ export type ProcessProtectedDataParams = {
    */
   app: AddressOrENS;
 
+  /**
+   * Address of an ERC734 whitelist contract authorized to access the protectedData and includes the current user address. This address will be used to search for granted accesses instead of the user address.
```

Comment:
```suggestion
   * Address of an ERC734 whitelist contract authorized to access the protectedData, including the current user address. This address will be used to search for granted accesses instead of the user address.
```